src/toil/job.py

```diff
@@ -3026,3 +3030,32 @@ def convertPromises(kwargs):
                 foundPromisedRequirement = True
         return foundPromisedRequirement
 
+
+class JobStoreFile(str):
```

Comment:
i was thinking of that and settled on this. I could change to that if you'd like